tools/ping/src/metrics.rs

```diff
@@ -0,0 +1,28 @@
+use near_o11y::metrics::{
+    exponential_buckets, try_create_histogram_vec, try_create_int_counter_vec, HistogramVec,
+    IntCounterVec,
+};
+use once_cell::sync::Lazy;
+
+pub(crate) static PONG_RECEIVED: Lazy<HistogramVec> = Lazy::new(|| {
```

Comment:
previously you had

```
prometheus::push_metrics(
                "near_ping",
                prometheus::labels! {
                    "chain_id".to_owned() => self.chain_id.clone(),
                },
```

but now the "chain_id" label isnt set anywhere. So that means that any current queries that reference chain_id (like we have set up now) will break. Does it make sense to add chain_id here? an alternative is to set it externally in the prometheus config. like for example none of the neard metrics actually have the chain_id label, but you can set it in the config w something like:

```
- source_labels:
      - __meta_*
      target_label: chain_id
```

Reply:
Added a label `chain_id`